src/Files.App/UserControls/SearchBox.xaml

```diff
@@ -58,7 +58,7 @@
 					<FontIcon
 						x:Name="EmptyIconGlyph"
 						x:Load="{x:Bind NeedsPlaceholderGlyph, Mode=OneWay}"
-						Glyph="&#xE7C3;" />
+						Glyph="{x:Bind Glyph, Mode=OneWay}" />
```

Comment:
Is it possible to use a converter here? It would be nice to set the glyphs directly from the xaml. Alternatively, a comment with the two possible glyphs would also work.